src/test/java/kz/toko/app/controller/UsersIT.java

```diff
@@ -79,7 +84,31 @@ void deleteAbsentUser() throws Exception {
     }
 
     @Test
-    @DisplayName("Should update only user name")
+    @DisplayName("Should create new user with default role")
+    void createUser() throws Exception {
+        final var createUserRequest = new CreateUserRequest()
+                .username("j_weak")
+                .firstName("John")
+                .lastName("Weak")
+                .password("password");
+
+        this.mockMvc.perform(
+                post("/users")
+                        .with(validJwtToken())
+                        .contentType(APPLICATION_JSON)
+                        .content(mapper.writeValueAsString(createUserRequest)))
+                .andDo(print())
+                .andExpect(status().isCreated())
+                .andExpect(jsonPath("$.id", notNullValue()))
+                .andExpect(jsonPath("$.username", is("j_weak")))
+                .andExpect(jsonPath("$.firstName", is("John")))
+                .andExpect(jsonPath("$.lastName", is("Weak")))
+                .andExpect(jsonPath("$.roles", hasItem(USER.getAuthority())));
```

Comment:
I think it would be nice also to check if this role is the only one assigned to the user by default.
You can check it by validating the roles count